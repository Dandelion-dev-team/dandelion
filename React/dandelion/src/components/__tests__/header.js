import React from "react"
import renderer from "react-test-renderer"
import Header from "../navigation/header"

jest.mock("../../auth");

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})