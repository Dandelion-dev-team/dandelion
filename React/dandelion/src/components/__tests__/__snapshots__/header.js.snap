// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <div
    className="inner-header"
  >
    <div
      className="logo"
    >
      <a
        exact={true}
        href="/"
      >
        <img
          src="test-file-stub"
        />
      </a>
    </div>
    <div
      className="navigation"
    >
      <nav>
        <a
          href="/data"
          onClick={[Function]}
        >
          Data
        </a>
        <a
          href="/map"
          onClick={[Function]}
        >
          Map
        </a>
        <a
          href="https://dandelion.scot/schools/growing-cubes/"
          onClick={[Function]}
          target="_blank"
        >
          About
        </a>
        <a
          href="/help"
          onClick={[Function]}
        >
          Help
        </a>
        <a
          href="/signin"
        >
          Sign In
        </a>
      </nav>
    </div>
    <div
      className="mobile-menu"
      onClick={[Function]}
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-i4bv87-MuiSvgIcon-root{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:1.5rem;}",
          }
        }
        data-emotion="css i4bv87-MuiSvgIcon-root"
      />
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium menu-icon css-i4bv87-MuiSvgIcon-root"
        data-testid="MenuIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </div>
  </div>
</header>
`;
